src/main/java/Barbie.java

```diff
@@ -0,0 +1,182 @@
+import types.Deadlines;
+import types.Party;
+import types.Task;
+import types.Todo;
+import exceptions.*;
```

Comment:
It would be better to avoid wildcard imports, rather you could list out the imports of the classes that are being used.